### PR TITLE
docs(prometheus_exporter sink): Update `flush_period_secs` docs

### PR DESCRIPTION
**Pull request by jszwedko**
This field took on more responsibility in
https://github.com/vectordotdev/vector/pull/9769. This PR updates the
doc to reflect its new semantics.

Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

**Review by lukesteensen — APPROVED** [review]

File: website/cue/reference/components/sinks/prometheus_exporter.cue [hunk 1]
```diff
@@ -81,7 +81,7 @@ components: sinks: prometheus_exporter: {
 		}
 		flush_period_secs: {
 			common:      false
-			description: "Time interval between [set](\(urls.vector_data_model)/metric#set) values are reset."
+			description: "Time interval on which metrics are flushed. On the flush interval, if a metric has not been seen since the last flush interval, it is considered expired and is remo. Be sure to configure this value higher than your client's scrape interval."
```
Comment (tobz) [inline website/cue/reference/components/sinks/prometheus_exporter.cue:84]: `is remo.` -> `is removed.`

**Review by tobz — CHANGES_REQUESTED** [review]

File: website/cue/reference/components/sinks/prometheus_exporter.cue (hunk 1 above)
Comment (jszwedko) [reply website/cue/reference/components/sinks/prometheus_exporter.cue:84]: Doh, thanks, not sure what happened there.

Comment (lukesteensen) [reply website/cue/reference/components/sinks/prometheus_exporter.cue:84]: Wow, my brain interpreted that just fine.

Comment (jszwedko) [reply website/cue/reference/components/sinks/prometheus_exporter.cue]: 😄 the brain is very good at filling in blanks.

**Review by tobz — APPROVED** [review]

**Pull request merged**